pkg/agent/plugin/nodeattestor/tpmdevid/devid_test.go

```diff
@@ -345,7 +345,7 @@ func TestConfigureWindows(t *testing.T) {
 	}
 }
 
-func TestAidAttestationFailiures(t *testing.T) {
```

Comment:
A little noise, but this test had been flaky when I started and the typo worked against me, so I decided to leave it a little better than I found it.